src/commands/show.ts

```diff
@@ -153,8 +153,21 @@ async function showProblemInternal(node: IProblem): Promise<void> {
                 leetCodeConfig.get<string>(`filePath.default.filename`) || genFileName(node, language),
             )
             .trim();
-
-        let finalPath: string = path.join(workspaceFolder, fileFolder, fileName);
+        
+        var i = fileName.indexOf('.');
+        const files: string[] = [fileName.slice(0, i), fileName.slice(i + 1)];
+        let id: string = node["id"];
```

Comment:
```javascript
let pad = "0000";
id  = pad.substring(0,  4 - id.length) + id
```
I think this way can drop these if and else if

Reply:
Nice! A good trick :)